docs/rules/no-replace-test-comments.md

```diff
@@ -0,0 +1,40 @@
+# no-replace-test-comments
+
+Ember devlopers using blueprints to generate classes should write real tests.
+
+Leaving the default test comment in place is a sign of a rushed class.
+
+## Rule Details
+
+This rule aims to nudge developers into writing more/better tests.
+
+It aims to do this by complaining at them early about a default test file.
+
+This will especially push TDD on repos with githooks that enforce lint early.
```

Comment:
Can you include `(test-driven development)` for readers who don't know this abbreviation?